docs/resources/unmanaged_user.md

```diff
@@ -3,10 +3,14 @@ subcategory: "User"
 ---
 # Artifactory Unmanaged User Resource
 
-Provides an Artifactory unmanaged user resource. This can be used to create and maintain Artifactory users.
-
+Provides an Artifactory unmanaged user resource. This can be used to create and manage Artifactory users.
+The password is required field by the [Artifactory API](https://www.jfrog.com/confluence/display/JFROG/Artifactory+REST+API#ArtifactoryRESTAPI-CreateorReplaceUser), but we made it optional in this resource to accommodate the scenario, when the password is not needed and will be reset by the actual user later.  
 When the optional attribute `password` is omitted, a random password is generated according to current Artifactory password policy.
 
+~> The generated password won't be stored in the TF state and can not be recovered. The user must reset the password to be able to log in. An admin can always generate the access key for the user as well. The password change won't trigger the state drift.
+
+~> This resource is an alias for `artifactory_user` [resource](https://registry.terraform.io/providers/jfrog/artifactory/latest/docs/resources/user), it is identical and was added for clarity and compatibility purpose. 
+
```

Comment:
Let's add a short sentence/paragraph to recommend *not* to use this resource unless their use cases fit this resource. And recommend our users to use `artifactory_managed_user` instead. Same for the original `artifactory_user` resource.